www/static/css-src/_home.scss

```diff
@@ -293,7 +293,7 @@ img#logo_top {
     .card {
             padding:8px;
             .card_inner {
-                background-color: darken(white, 85%);
```

Comment:
Here too, since this is happening on a static colour `white` we should just figure out what the hex value is and use it directly or store it directly in a variable instead of needing a colour function here.